house/views.py

```diff
@@ -1,7 +1,13 @@
+from house.constants import HOME_PAGE_ROUTE, HOUSE_PAGE_ROUTE
+from .models import House
+from expenses.models import Expenses
+from django.shortcuts import render, get_object_or_404
 
 
 def home_page(request):
-    raise NotImplementedError
+    # this line is needed till the login of the mine page will create and merge
+    print(House.objects.all().order_by("?").first().house_id)
```

Comment:
Please remove prints before merging it into master
You did it to take a random house id and then go to its URL?

Reply:
as I wrote - I'll remove the print line after @LiorKGOW will do the login view.